content/mozdev/revert.rst

```diff
@@ -0,0 +1,32 @@
+public: yes
```

Comment:
I, uh, still don't understand having duplicate files for these. But I assume there must be a good reason -- in which case should there be a post for `list-markers` too?

Reply:
I'm happy to talk it through with you, if that would help?

- in order to show up on the blog, we need to have a "post"
- in order for it to show up in the speaking list, we need to have a "page"

That is an rstBlog distinction - in 11ty we  could do both with a single file. I can try and find a workaround in the meantime, but I'd rather spend  that time getting us moved off rstBlog. 

List-markers already has a post - just not all-to-itself: it's the mozdev announcement post.